include/seqan3/alphabet/nucleotide/concept.hpp

```diff
@@ -43,10 +43,10 @@ struct complement_fn
         {
             { impl(priority_tag<2>{}, nucl) };
             requires noexcept(impl(priority_tag<2>{}, nucl));
-            requires std::same_as<nucleotide_t, decltype(impl(priority_tag<2>{}, nucl))>;
+            { impl(priority_tag<2>{}, impl(priority_tag<2>{}, nucl)) }; // you can take the complement again
```

Comment:
We need a back log item to analyse why `requires std::common_with<decltype(impl(priority_tag<2>{}, nucl)), nucleotide_t>;` is not working correctly. This is rather due to some error on our side I believe.

Reply:
Also please add the following: `requires alphabet<decltype(impl(priority_tag<2>{}, nucl))>;` to at least gurantee, that I don't return some uncomplete alphabet type that just offers a complement function.